doc/api/tls.md

```diff
@@ -445,6 +445,47 @@ The first three are enabled by default. The two `CCM`-based suites are supported
 by TLSv1.3 because they may be more performant on constrained systems, but they
 are not enabled by default since they offer less security.
 
+## OpenSSL Security Level
+
+The OpenSSL library enforces security levels to control the minimum acceptable
+level of security for cryptographic operations. OpenSSL's security levels range
+from 0 to 5, with each level imposing stricter security requirements. The default
+security level is 1, which is generally suitable for most modern applications.
+However, some legacy features and protocols, such as TLSv1, require a lower
+security level (`SECLEVEL=0`) to function properly. For more detailed information,
+please refer to the [OpenSSL documentation on security levels][].
+
+### Setting Security Levels
+
+To adjust the security level in your Node.js application, you can include `@SECLEVEL=X`
+within a cipher string, where `X` is the desired security level. For example,
+to set the security level to 0 while using the default OpenSSL cipher list, you could use:
+
+```js
+const tls = require('tls');
```

Comment:
nit: `node:tls` is preferred (AFAIK)
nit 2: could you include both a CJS and ESM example?

Reply:
+1 on `node:tls`.

I'm not sure an ESM example is required here (or at least, it's a separate topic) as all other examples in here currently use only CJS, and all other docs pages I've just quickly checked do the same. There's probably a wider question of what our docs policy is on this that might be interesting, but let's handle that elsewhere :smile: